hicommonwealth/commonwealth
# Add webpack define for magic publishable key.

To test:
1. Add a `console.log(process.env.MAGIC_PUBLISHABLE_KEY)` at the top of `login.ts` on the client.
2. Remove your .env value for MAGIC_PUBLISHABLE_KEY. Run `yarn start`, load the site, and ensure that you see a console log of the default value in `webpack.common.js`.
3. Set the value in your .env file, kill and re-run `yarn start`, ensure it prints the set value.

We can use a similar strategy to expose env vars on the client side moving forward.
